addons/account/models/chart_template.py

```diff
@@ -926,7 +926,21 @@ def _get_tag_mapper(self, template_code):
             ('applicability', '=', 'taxes'),
             ('country_id', '=', self._get_chart_template_mapping()[template_code]['country_id']),
         ])}
-        return lambda *args: [tags[re.sub(r'\s+', ' ', x.strip())] if not re.match(r"^\w+\.\w+$", x) else x for x in args]
+
+        def _get_mapping(*args):
+            res = []
+            for tag in args:
+                if re.match(r"^\w+\.\w+$", tag):
+                    # unsigned tag => comes from a template, doesn't need to be mapped
+                    res.append(tag)
+                else:
+                    format_tag = re.sub(r'\s+', ' ', tag.strip())
+                    mapped_tag = tags.get(format_tag)
+                    if not mapped_tag:
+                        raise UserError(_('Could not find tag "%s". You should probably update your localization module first.', format_tag))
```

Comment:
Need to add the entry in the pot

Is the `Could not find tag "%s"` part really useful for the end user? (or even the support)

Reply:
Well, I find it useful else I wouldn't have put it :smirk: 
I'd say if you click the reload button you arguably already have some experience with your Odoo accounting and would like to know what the problem is in more details.
But I can live without it if you prefer to remove it.